src/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -6,7 +6,7 @@ const rule = rules[ruleName]
 
 testRule(rule, {
   ruleName,
-  config: [undefined],
+  config: [true],
```

Comment:
You'll want to avoid changing these, as this was done in bulk on the `v7` branch. This was cause merge conflicts.